sdk/compute/arm-compute/src/operations/restorePointCollections.ts

```diff
@@ -268,6 +312,7 @@ export class RestorePointCollectionsImpl implements RestorePointCollections {
     const poller = await createHttpPoller<void, OperationState<void>>(lro, {
       restoreFrom: options?.resumeFrom,
       intervalInMs: options?.updateIntervalInMs,
+      resourceLocationConfig: "location",
```

Comment:
add x-ms-long-running-operation-options for this operation in generated swagger     
```  
"x-ms-long-running-operation-options": {
          "final-state-via": "location"
        },
```

Reply:
Yes, it is expected change. Is there any problem for you to add this?

Reply:
offline confirmed with @MaryGao this should be fine for us.